src/hotspot/share/ci/ciMethodData.hpp

```diff
@@ -511,17 +517,28 @@ class ciMethodData : public ciMetadata {
   ciProfileData* first_data() { return data_at(first_di()); }
   ciProfileData* next_data(ciProfileData* current);
   DataLayout* next_data_layout(DataLayout* current);
+  DataLayout* next_extra_data_layout(DataLayout* current);
   bool is_valid(ciProfileData* current) { return current != nullptr; }
   bool is_valid(DataLayout* current)    { return current != nullptr; }
 
+  // pointers to sections in _data
+  // NOTE: these may be called before ciMethodData::load_data (is that a bug?).
```

Comment:
I don't think we should add a "is that a bug?" comment here but either investigate right away or file a follow-up RFE to keep track of that. I don't think it's a bug though.

Reply:
Right. This is a leftover personal note that I added during development. Will remove.